docs/navigation/overlay/component-context.md

```diff
@@ -0,0 +1,69 @@
+# How to initialize Child Overlay with custom ComponentContext 
+
+In order to pass this `AppComponentContext` to child overlay components, make an extension function on the `AppComponentContext` interface. This custom extension function will initialize the `Child Overlay` and provide child `AppComponentContext`.
```

Comment:
It seems that the underlying context of what are "custom component context" and `AppComponentContext` is missing here, if you just jump to this section without reading the `Custom ComponentContext` page. Maybe add something like the following?

> See [Custom ComponentContext](...) page for more information about creating custom `AppComponentContext`.

Reply:
Nice catch I overseen this, thanks!